code/mod_table/mod_table.cpp

```diff
@@ -397,6 +398,16 @@ void parse_mod_table(const char *filename)
 			stuff_boolean(&Neb_affects_fireballs);
 		}
 
+		if (optional_string("$Shadow Cascade Distances:")) {
+			float dis[4];
+			stuff_float_list(dis, 4);
+			if ((dis[0] >= 0) && (dis[1] >= 0) && (dis[2] >= 0) && (dis[3] >= 0)) {
+				Shadow_distances = std::make_tuple(static_cast<float>(dis[0]), static_cast<float>(dis[1]), static_cast<float>(dis[2]), static_cast<float>(dis[3]));
```

Comment:
Is the cast to float necessary here? The original array is already a float so I would guess not.